### PR TITLE
Add filesystem koji task id to labels

### PR DESCRIPTION
**Pull request by bfontecc007**
This is related to story 3955. The idea here is to add the koji filestystem task id to the labels (in the orchestrator as well as worker).

Signed-off-by: Bret Fontecchio <bfontecc@redhat.com>

### CODE REVIEW

File: atomic_reactor/plugins/exit_store_metadata_in_osv3.py [hunk 1]
```diff
@@ -62,6 +64,10 @@ def get_config_map(self):
 
         return annotations
 
+    def get_filesystem_koji_task_id(self):
+        res = self.get_result(self.workflow.prebuild_results.get(PLUGIN_ADD_FILESYSTEM_KEY, ''))
```
Comment (lcarva) [inline atomic_reactor/plugins/exit_store_metadata_in_osv3.py]: Don't think you need the default empty string here. Omit second parameter to `get` letting it default to `None`.

File: atomic_reactor/plugins/exit_store_metadata_in_osv3.py [hunk 2]
```diff
@@ -62,6 +64,10 @@ def get_config_map(self):
...
+        return graceful_chain_get(res, 'filesystem-koji-task-id') or ''
```
Comment (lcarva) [inline atomic_reactor/plugins/exit_store_metadata_in_osv3.py]: Unnecessary `or ''` here.

File: tests/plugins/test_store_metadata.py [hunk 3]
```diff
@@ -288,6 +289,40 @@ def test_metadata_plugin(tmpdir, br_annotations, expected_br_annotations,
...
+    )
+    output = runner.run()
+    labels = output[StoreMetadataInOSv3Plugin.key]["labels"]
+
+    if 'filesystem-koji-task-id' in res:
+        assert 'filesystem-koji-task-id' in labels
+        assert labels['filesystem-koji-task-id'] == 'example-fs-taskid'
+
+
+
```
Comment (twaugh) [inline tests/plugins/test_store_metadata.py]: Just need 2 blank lines here, not 3.

File: tests/plugins/test_store_metadata.py [hunk 4]
```diff
@@ -288,6 +289,40 @@ def test_metadata_plugin(tmpdir, br_annotations, expected_br_annotations,
...
+            }
+        }]
+    )
+    output = runner.run()
+    labels = output[StoreMetadataInOSv3Plugin.key]["labels"]
+
+    if 'filesystem-koji-task-id' in res:
+        assert 'filesystem-koji-task-id' in labels
+        assert labels['filesystem-koji-task-id'] == 'example-fs-taskid'
+
```
Comment (twaugh) [inline tests/plugins/test_store_metadata.py]: We don't assert anything if `'filesystem-koji-task-id' not in res`. Are we just making sure the plugin copes with it OK? Maybe we should assert that no 'filesystem-koji-task-id' label is added.

File: tests/plugins/test_store_metadata.py [hunk 5]
```diff
@@ -288,6 +289,40 @@ def test_metadata_plugin(tmpdir, br_annotations, expected_br_annotations,
         assert json.loads(annotations['media-types']) == expected_pulp_pull_results
 
 
+@pytest.mark.parametrize(('res'), (
+    {
+        'filesystem-koji-task-id': 'example-fs-taskid',
+        'base-image-id': 'foobar'
```
Comment (twaugh) [inline tests/plugins/test_store_metadata.py]: Nitpick: I usually put a comma after each entry in a sequence or dict when they're on separate lines like this, even the last, to make diffs easier to read if we add/remove/swap lines afterwards.

File: tests/plugins/test_store_metadata.py [hunk 6]
```diff
@@ -288,6 +289,41 @@ def test_metadata_plugin(tmpdir, br_annotations, expected_br_annotations,
...
+            }
+        }]
+    )
+    output = runner.run()
+    labels = output[StoreMetadataInOSv3Plugin.key]["labels"]
+
+    if 'filesystem-koji-task-id' in res:
+        assert 'filesystem-koji-task-id' in labels
+        assert labels['filesystem-koji-task-id'] == 'example-fs-taskid'
+    if 'filesystem-koji-task-id' not in res:
```
Comment (twaugh) [inline tests/plugins/test_store_metadata.py]: nit: could just use `else:` here

Comment (vrutkovs) [inline tests/plugins/test_store_metadata.py]: +1

**Comment by twaugh** [comment]
Draft release notes updated.

**Pull request merged**